app/index.js

```diff
@@ -41,8 +44,18 @@ DB.connect().catch((error) => {
   .then(() => express(app))
   .then(() => routes.registerRoutes(app))
   .then(() => AddonManager.init(app, server, io))
+  .then(() => {
+    if (cluster.isMaster) {
```

Comment:
this could be simplify a bit:
```
let pending = [
  AddonManager.readAddons(),
  cluster.IsMaster ? AddonManager.installAddons() : Promise.resolve()
];
return Promise.series(pending);
```